docs/user/intro.rst

```diff
@@ -1,263 +0,0 @@
-Most such Parameters have a simple numeric value, but the value can be a string or other data type if necessary.
-If a settable Parameter is also gettable, getting it typically just reads back what was previously set, through QCoDeS or by some other means,
-but there can be differences due to rounding, clipping, feedback loops, etc.
-Note that setting a Parameter of a metainstrument may involve setting several lower-level Parameters of the underlying Instruments,
-or even getting the values of other Parameters to inform the value(s) to set.
-
-A Parameter that is only gettable typically represents a single measurement command or sequence.
-
-The value of such a Parameter may be of many types:
-  - A single numeric value, such as a voltage measurement
-  - A string that represents a discrete instrument setting, such as the orientation of a vector
-  - Multiple related values, such as the magnitude and phase or Cartesian components of a vector
-  - A sequence of values, such as a sampled waveform or a power spectrum
-  - Multiple sequences of values, such as waveforms sampled on multiple channels
-  - Any other shape that appropriately represents a characteristic of the Instrument.
-
-When a RemoteInstrument is created, the Parameters contained in the Instrument are mirrored as RemoteParameters,
-which connect to the original Parameter via the associated InstrumentServer.
-
-**Computed Measurements**
-
-In some cases the measurement value of interest is computed based on values read from more than one instrument.
-For example, you might want to track the power dissipation of a component, computed by multiplying the outputs from a voltmeter and an ammeter.
-QCoDeS allows you to define a Parameter that represents the result of such a computation and include it as part of your experimental results.
-
-A Parameter defined in this way is not associated with an Instrument.
-It is a Python object that encapsulates the computation to be performed and references the underlying Parameters it uses.
-It is gettable, but not settable.
-
-**Interdependent Settings**
-
-In some experiments, control values for one or more Instruments must be set together in order to maintain a condition.
-For example, you might want to measure the behavior of a component at different voltage levels,
-but always keeping the available power within a fixed bound.
-You can define a Parameter that gets initialized with the maximum power to allow, such that setting the Parameter value
-results in setting the voltage to the passed-in value and also adjusting the supplied current appropriately.
-
-Similarly to a Parameter that computes a measurement result, this type of Parameter is not associated with an Instrument.
-It encapsulates the way that the single setting impacts the related settings and references the underlying Parameters, one for each setting.
-It is settable, and may be gettable.
-
-
-Loop
```

Comment:
i briefly looked at the legacy exmples section and did not see there this high-level description of Loop and Measure (maybe i didn't look to well :) ). so perhaps it makes sense to just move the content that is below this line into a separate notebook under the "legacy examples"?

Reply:
It is there (e.g. https://qcodes.github.io/Qcodes/user/intro.html#loop https://qcodes.github.io/Qcodes/user/intro.html#measure). 
We can move this and any other legacy text to the legacy example page sure. I just don't see the purpose of maintaining it further as a 'user guide'.

Reply:
> I just don't see the purpose of maintaining it further as a 'user guide'.

that i agree :)

> We can move this and any other legacy text to the legacy example page sure

let's do this then :)

Reply:
I should have started at the top and moved down, yes.